packages/maps/src/components/marker-cluster/MapMarkerCluster.tsx

```diff
@@ -79,18 +75,18 @@ const MapMarkerCluster: React.FC<
                   longitude,
                   children: clusterView,
                   onPress,
-                  tracksViewChanges,
+                  tracksViewChanges:
+                    clusterView.type === DefaultMapMarkerClusterView
+                      ? false
+                      : clusterView.props.tracksViewChanges,
```

Comment:
This would result in it always being true if not the `DefaultMapMarkerClusterView`, we want to instead use the clusterView.props.tracksViewChanges. Or am I misunderstanding?

Reply:
It is tricky 😄. I kept it as is because it's easier to understand what's going on.